### PR TITLE
Release 1.1.0rc1

### PR DESCRIPTION
**Pull request by jeremiedbb**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jeremiedbb** [comment]
@conda-forge-admin, please rerender

**Comment by jeremiedbb** [comment]
@conda-forge-admin, please rerender

**Comment by jeremiedbb** [comment]
@conda-forge-admin, please rerender

**Comment by thomasjpfan** [comment]
`clear_all_globals` was removed in https://github.com/scipy/scipy/commit/bf73d92e9ff02b25543d45c2bf30bd04fe3c47fd which was released in 1.8.0.

From the failing job, it looks like the arch64 environment for python 3.8 is using SciPy 1.5.3 in one environment and 1.8.0 in another one.

**Comment by jeremiedbb** [comment]
I saw that but it makes no sense to me why it happens for this build and not for the 2 other aarch64 builds :/

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,63 +34,95 @@ requirements:
...
-    - cython >=0.28.5
+    - setuptools <60.0
+    - cython >=0.29.24
     - libcblas
     - numpy
-    - scipy >=0.17.0
-    - joblib >=0.11
-    - threadpoolctl
+    - scipy >=1.3.2  # [not aarch64]
+    - scipy >=1.8  # [aarch64]
```
Comment (thomasjpfan) [inline recipe/meta.yaml:42]: I'm okay with this workaround.

**Review by thomasjpfan — APPROVED** [review]
Thank you for the fix!

**Comment by jeremiedbb** [comment]
Let's merge then

**Pull request merged**